cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/dslHelp.groovy

```diff
@@ -0,0 +1,27 @@
+package org.jenkinsci.plugins.workflow.cps.Snippetizer
+
+
+def l = namespace(lib.LayoutTagLib)
+def st = namespace("jelly:stapler")
+
+l.layout(title:_("Jenkins Workflow Reference")) {
+
+    st.include(page: "sidepanel", it: app)
+    l.main_panel {
+        p {
+            a(href: "${app.rootUrl}${my.GDSL_URL}", target: "_blank") {
```

Comment:
:bug: must use `rootURL` in pages served from Jenkins. Never use `app.rootUrl` except when generating HTML to be sent elsewhere. (I thought I mentioned this already? [wiki](https://wiki.jenkins-ci.org/display/JENKINS/Hyperlinks+in+HTML))